internal/export/distro/registrations.go

```diff
@@ -15,18 +15,31 @@ package distro
 
 import (
 	"fmt"
-	"net/http"
-	"time"
-
-	export "github.com/edgexfoundry/edgex-go/export"
-
+	"github.com/edgexfoundry/edgex-go/internal/export"
+	"github.com/edgexfoundry/edgex-go/internal/export/interfaces"
 	"github.com/edgexfoundry/edgex-go/pkg/models"
+	"net/http"
 
 	"go.uber.org/zap"
 )
 
 var registrationChanges chan export.NotifyUpdate = make(chan export.NotifyUpdate, 2)
 
+// RegistrationInfo - registration info
+type registrationInfo struct {
+	registration export.Registration
+	format       interfaces.Formatter
+	compression  interfaces.Transformer
+	encrypt      interfaces.Transformer
+	sender       interfaces.Sender
+	filter       []interfaces.Filterer
+
+	chRegistration chan *export.Registration
+	chEvent        chan *models.Event
+
+	deleteMe bool
```

Comment:
I would prefer just `delete` or `deleteFlag` for the name of this var. Looks confusing, like the var itself should be deleted :).